.travis.yml

```diff
@@ -7,17 +7,18 @@ addons:
   postgresql: "9.3"
 env:
   matrix:
-    - MODULE=!querydsl-scala
-    - MODULE=querydsl-scala
+    - MODULE=!querydsl-scala PROFILES=all,travis,examples
+    - MODULE=!querydsl-jpa-codegen PROFILES=jpa,hibernate5,travis,examples
```

Comment:
Or hmm, I forgot that it is a `--projects` pattern.
It seems to work nevermind.